ApiDoctor.Console/Program.cs

```diff
@@ -2139,7 +2165,7 @@ private static void InjectSnippetIntoFile(MethodDefinition method, string codeSn
                         if (originalFileContents[currentIndex].Contains($"(#tab/{codeFenceString})"))
                         {
                             originalFileContents[currentIndex] = $"# [{language}](#tab/{codeFenceString})";
-                            originalFileContents[currentIndex + 1] = $"[!INCLUDE [sample-code](../{relativePathSnippetsFolder}{snippetFileName})]";//update include link. Just in case.
+                            originalFileContents[currentIndex + 1] = $"[!INCLUDE [sample-code](../{ReplaceWindowsByLinuxPathSepartors(relativePathSnippetsFolder)}{snippetFileName})]";//update include link. Just in case.
```

Comment:
```suggestion
                            originalFileContents[currentIndex + 1] = $"[!INCLUDE [sample-code](../{ReplaceWindowsByLinuxPathSeparators(relativePathSnippetsFolder)}{snippetFileName})]";//update include link. Just in case.
```